<launch>
    <node pkg="tof1_driver" exec="tof_node" name="tof_node" namespace="tof1_driver">
        <param name="camera_params" value="package://tof1_driver/cam_param.yaml"/>
    </node>

    <include file="$(find-pkg-share rtabmap_launch)/launch/rtabmap.launch.py">
        <arg name="args" value="--delete_db_on_start" />
        <arg name="depth" value="false" />
        <arg name="subscribe_scan_cloud" value="true" />
        <arg name="subscribe_rgb" value="false" />
        <arg name="subscribe_depth" value="false" />
        <arg name="scan_cloud_topic" value="/tof1_driver/camera/depth/pcl_depth" />
        <arg name="camera_info_topic" value="/tof1_driver/camera/depth/cam_info" />
        <arg name="icp_odometry" value="true" />
        <arg name="frame_id" value="map" />
        <arg name="approx_sync" value="false" />
        <arg name="qos" value="1" />
        <arg name="rviz" value="true" />
        <arg name="rtabmap_viz" value="false" />
    </include>
</launch>